src/openms/source/CHEMISTRY/NASequence.cpp

```diff
@@ -211,6 +211,7 @@ namespace OpenMS
     {
       local_five_prime = five_prime_->getFormula() - H_form;
     }
+    cout << local_five_prime << endl;
```

Comment:
Remove ?

Reply:
oof, forgetting to save the buffer before committing strikes again.